composer.json

```diff
@@ -23,6 +23,7 @@
     "illuminate/support": "~5.5.0|~5.6.0|~5.7.0|~5.8.0|^6.0|^7.0|^8.0|^9.0"
   },
   "require-dev": {
+    "friendsofphp/php-cs-fixer": "^3.15",
```

Comment:
Allows us to run the fixer locally, though not really necessary as we already run it in the Github Action.

Could be useful when trying out other settings.

Reply:
```suggestion
```